sky/utils/command_runner.py

```diff
@@ -260,6 +262,7 @@ def run(
             stream_logs: bool = True,
             ssh_mode: SshMode = SshMode.NON_INTERACTIVE,
             separate_stderr: bool = False,
+            connection_timeout: int = 30,
```

Comment:
ditto

also, None here so defaults are only recorded in one place? https://github.com/skypilot-org/skypilot/blob/230fb22cf48076a00ad559307f7b878b9bbdc945/sky/utils/command_runner.py#L44